src/components/Property.astro

```diff
@@ -17,9 +17,10 @@ const link = customDataTypes.includes(formattedType) ? '/api/data-types#' + form
       {name}
     </dd>
     <dt class="sr-only">Type</dt>
-    <dd class="font-mono text-xs text-zinc-400 dark:text-zinc-500">
+    <dd>
```

Comment:
@alexBinns each <dd> tag should have a <dt> with it.
If we don't want it visible we can give it the class `sr-only`.
You can also make the conditional logic simpler.

Example:
```
{ required && 
  <dt class="sr-only">Required</dt>
  <dd><Badge type="required"></dd>
}
```